code/web/src/modules/user/api/actions.js

```diff
@@ -117,3 +118,26 @@ export function getGenders() {
     }))
   }
 }
+
+// Update a user information
+export function updateUserInformation(userDetails) {
+  return dispatch => {
+    return axios.post(routeApi, mutation({
+      operation: 'editUser',
+      variables: userDetails,
+      fields: ['id', 'name', 'email', 'shippingAddress', 'description']
+    }))
+    .then(respone => {
+      editProfileSetLocalStorageAndCookie(userDetails)
+      dispatch({
+        type: UPDATE_USER,
+        userDetails
+      })
+    })
+  }
+}
```

Comment:
This is the reducer that unfortunately seems to not be working. In the console, receive a 400 error saying it is a bad request.